R/zzz.R

```diff
@@ -13,6 +13,7 @@
   PythonEmbedInR::pyImport("synapseclient")
   PythonEmbedInR::pySet("synapserVersion", sprintf("synapser/%s ", utils::packageVersion("synapser")))
   PythonEmbedInR::pyExec("synapseclient.USER_AGENT['User-Agent'] = synapserVersion + synapseclient.USER_AGENT['User-Agent']")
+  PythonEmbedInR::pyExec("synapseclient.config.single_threaded = True")
```

Comment:
please write a test that invokes the Python pool provider. It should break when multithreading is enabled but pass when disabled.